src/utilities.jl

```diff
@@ -369,7 +369,7 @@ function arguments(m::Method)
     local template = get_method_source(m)
     if isdefined(template, :slotnames)
         local args = map(template.slotnames[1:nargs(m)]) do arg
-            arg === Symbol("#unused#") ? "?" : arg
+            arg === Symbol("#unused#") || arg === Symbol("") ? "_" : arg
```

Comment:
I am not completely sure here, but it seems that `arg == Symbol("")` happens when the original argument as actually missing. I feel that in those cases (e.g. `::Type{T}` being a common example) we probably shouldn't be putting a `_` into the signature?

Reply:
Even I'm not sure what people would like here. I went with this because the use of underscores is consistent, but after reading your comment and looking at the signature I'm not sure users would value this. I'll change it back like you suggested.

Reply:
Resolved in 719e899.